app/main/views/briefs.py

```diff
@@ -154,10 +154,13 @@ def list_briefs():
             Lot.slug.in_(lot_slug.strip() for lot_slug in request.args["lot"].split(","))
         ))
 
+    response_headers = {"X-Compression-Safe": "0"}
     if request.args.get('status'):
-        briefs = briefs.has_statuses(
-            *(status.strip() for status in request.args['status'].split(','))
-        )
+        statuses = frozenset(status.strip() for status in request.args['status'].split(','))
+        briefs = briefs.has_statuses(*statuses)
```

Comment:
What are we inferring from this condition? The likelihood of it being an indexing job?

If so I think it would be safer to have a parameter on the view which would allow the client to decide whether they expect the response compressed.

`allow_compression` then add the compression header regardless of the request 🤷‍♂ In fact; that would be a good thing for all our api views to have.

Reply:
> What are we inferring from this condition? The likelihood of it being an indexing job?

No, we're actually deciding based on what we think is safe to output. Drafts aren't safe because they're not meant to be public. Withdrawn briefs aren't public - they were withdrawn for a reason. User data isn't generally public. So if the endpoint can be sure that none of this data is going to be output it'll allow it to be compressed.

The client can always opt out by not including `Accept-Encoding gzip`.

Reply:
However, the point stands that this could do with a comment or two.

Reply:
👍 Ah fairs, it's _extra_ security. Sounds good to me